src/platform/tests/Response/StreamResponseTest.php

```diff
@@ -15,9 +15,9 @@
 use PHPUnit\Framework\Attributes\Small;
 use PHPUnit\Framework\Attributes\Test;
 use PHPUnit\Framework\TestCase;
-use Symfony\AI\Platform\Response\StreamResponse;
+use Symfony\AI\Platform\Result\StreamResult;
 
-#[CoversClass(StreamResponse::class)]
+#[CoversClass(StreamResult::class)]
 #[Small]
 final class StreamResponseTest extends TestCase
 {
```

Comment:
class name correct?

Reply:
noo, please don't review - it's incomplete

Reply:
now :tada: